design/authenticators/authn_oidc/oidc_configuration_enhancements.md

```diff
@@ -0,0 +1,228 @@
+# OIDC Authenticator Configuration Improvements
+
+## Issue Description
+There have been multiple instances where users of the OIDC authenticator have experienced limitations resulting 
+from our use of the third-party [OpenIDConnect gem](https://github.com/nov/openid_connect). The issues mainly stem 
+from the gem limiting changes to the HTTP config used for outgoing connections. So far the following problems have 
+been identified:
+- Unable to configure CA certs from the authenticator config in Conjur
+  - The workaround for custom CA certs is to add them to the Conjur container OpenSSL truststore (`/etc/ssl/certs`)
+- The underlying HTTP client does not honor the `HTTPS_PROXY` environment variable
+- Limited ability to debug OIDC-related HTTP errors
+
+## Solution
+It would be more consistent with other authenticator configs if authn-oidc were to support a `ca-cert` variable 
+in the authenticator policy. This value would be used to inform the HTTP client which CA cert(s) to use to verify 
```

Comment:
Is this a single CA cert or Chain?  How would you handle multiple certs - would these need to be concatenated together?

Reply:
It should support both, which we can do via a pre-existing [parse_certs](https://github.com/cyberark/conjur/blob/48e95904a5ee8cda1503db9f5744ff6eefcecbdb/app/domain/conjur/cert_utils.rb#L11C10-L11C10) method. In the policy they would be concatenated together, then Conjur would separate them into individual certs for the purpose of adding them to the truststore.